wcivf/apps/elections/management/commands/import_ballots.py

```diff
@@ -39,6 +39,15 @@ def populate_any_non_by_elections_field(self):
             election.any_non_by_elections = any_non_by_elections
             election.save()
 
+    def delete_deleted_elections(self):
```

Comment:
I think this should be decorated with a `@transaction.atomic`, just in case something goes wrong.